timelinejs3/timelinejs3.d.ts

```diff
@@ -0,0 +1,351 @@
+// Type definitions for timelinejs3
```

Comment:
I'm using these definitions with 3.4.1, which is the latest version. Should I add that in the comments somewhere?